cdap-ui/app/cdap/components/SortableStickyGrid/index.js

```diff
@@ -48,7 +48,7 @@ export default class SortableStickyGrid extends Component {
     gridHeaders: PropTypes.arrayOf(
       PropTypes.shape({
         label: PropTypes.string,
-        property: PropTypes.string,
+        property: PropTypes.oneOfType([PropTypes.string, PropTypes.func]),
```

Comment:
Should this be in this PR?

Reply:
it's not related to the fix, however this was throwing validation error in the same page, since there's a table to show the list of runs on the dashboard page